assets/src/scripts/components/hydrograph/store.js

```diff
@@ -1,43 +1,57 @@
+const merge = require('lodash/merge');
```

Comment:
I don't think this got added to package.json.

Reply:
Oh, must be a dependency of another package we use. Will add.